### ISSUE TITLE
How to display train_loss and val_loss in each step 

### ISSUE THREAD

**Issue by QuickLearner171998** [tensorflow/models#6945]
I am using model_main.py from tensorflow /models, I am able to get the train loss, however I want to display both train_loss and val_loss in each step. Please suggest me the changes to be made in the relevant files. 

Thanks

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
What is the top-level directory of the model you are using
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce

**Comment by QuickLearner171998** [comment]
> Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
> What is the top-level directory of the model you are using
> Have I written custom code
> OS Platform and Distribution
> TensorFlow installed from
> TensorFlow version
> Bazel version
> CUDA/cuDNN version
> GPU model and memory
> Exact command to reproduce

I have kept everything same and am using model_main.py and my config file is faster_rcnn_inception_v2_pets.config, I and things are working fine. But in addition to that I want to print val_loss in each step. Also please tell me which file is actually printing the train loss.

Thanks

**Comment by tensorflowbutler** [comment]
Hi There,
 We are checking to see if you still need help on this, as this seems to be an old issue. Please update this issue with the latest information, code snippet to reproduce your issue and error you are seeing.
 If we don't hear from you in the next 7 days, this issue will be closed automatically. If you don't need help on this issue any more, please consider closing this.
